cwl/src/main/scala/wdl4s/cwl/WorkflowStep.scala

```diff
@@ -220,14 +223,21 @@ case class WorkflowStepOutput(id: String)
 
 object WorkflowStep {
 
-  private [cwl] object WorkflowStepInputFold {
-    private [cwl] def emptyRight = WorkflowStepInputFold(Map.empty, Set.empty).asRight[NonEmptyList[String]]
+  implicit val workflowStepInputFoldMonoid: Monoid[WorkflowStepInputFold] = new Monoid[WorkflowStepInputFold] {
+    override def empty: WorkflowStepInputFold = WorkflowStepInputFold()
+    override def combine(x: WorkflowStepInputFold, y: WorkflowStepInputFold): WorkflowStepInputFold = {
+      WorkflowStepInputFold(
+        stepInputMapping = x.stepInputMapping ++ y.stepInputMapping,
+        generatedNodes = x.generatedNodes ++ y.generatedNodes
+      )
```

Comment:
It would be nice to have a comment here about why you had to write this out manually so nobody comes along later and has to figure out the hard way why they can't use kittens to do this.